integration/src/test/resources/integration/view_capabilities.feature

```diff
@@ -0,0 +1,10 @@
+Feature: View Capability a role belongs to
+  Scenario Outline: want to be able to see what capability each role falls within
+    Given The Kainos employee is on the capabilities page
+    Then The client clicks on the <caps> capability
+    And capability has <results> jobs
```

Comment:
There seems to be a disjoint between the wording of the scenario and the steps. You're only checking how many roles are in each capability - so either the outline should be reworded, or the steps changed to check for specific roles. You could have three rows with Engineering and the second column be each role you expect in there.

Note that these tests are dependant on the database contents - so it would break if the database was changed.